app/models/micropost.rb

```diff
@@ -0,0 +1,18 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  scope :by_date -> {order created_at: :desc}
```

Comment:
Please place the missing comma(,), else it will throw error
`scope :by_date, -> {order created_at: :desc}`